neighboorhood/src/componenets/App.jsx

```diff
@@ -24,7 +47,7 @@ class App extends React.Component {
               </div>
             </div>
             <div>
-              <img src="https://cdn.acidcow.com/pics/20170117/pretty_girls_02.jpg" alt="" />
+              <img src="https://marriedbiography.com/wp-content/uploads/2017/03/Jessica-Burciaga.jpg" alt="" />
```

Comment:
does this hardcode in some of the host/owner information since we don't have full dummy data for this app?

Reply:
I didnt have enough time to develop dummy data for the general and neighborhood. It took longer than expected. Something i can implement in the future if theres enough time